storybook/src/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Select } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Components/Forms/Select',
+  component: Select,
+  args: {
+    invalid: false,
+    required: false,
```

Comment:
`required` is a native attribute, that doesn't change the styling in our implementation. I don't think we really need to have it as a control.